_dev/src/components/commons/select-country.vue

```diff
@@ -78,6 +78,11 @@ export default {
       required: false,
       default: false,
     },
+    multipleCountries: {
+      type: Boolean,
+      required: true,
```

Comment:
Could be optional, so you only add the prop `:multiple-countries="true"` only when you want multiple target countries, the other ones remain untouched.